Dockerfile

```diff
@@ -13,5 +13,8 @@ FROM openjdk:11-jdk-slim
 ARG path=/usr/app
 WORKDIR ${path}
 COPY --from=build ${path}/target/*.jar ${path}/app.jar
+
 EXPOSE 8080
-ENTRYPOINT ["java","-jar","app.jar"]
\ No newline at end of file
+
+#ENTRYPOINT ["java","-jar","app.jar", "-Dserver.port=$PORT", "-jar target/*.jar"]
```

Comment:
If `ENTRYPOINT` is not used, remove it at all

Reply:
done